From 687d3c3ed470ab554975883e1bba3dcd2cc99e79 Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Tue, 26 Sep 2023 21:29:52 +0530
Subject: [PATCH] OMX: Remove support for prebuilt ac4 decoder

Requires proprietary Dolby changes in libstagefright.

This partially reverts commit 01090fa125c2993654e18e024ddf7cef87bca30e.

Change-Id: Ifdf36ad253155e77ba14d0624612bf6be6711ca6
---
 media/libstagefright/omx/SoftOMXPlugin.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/media/libstagefright/omx/SoftOMXPlugin.cpp b/media/libstagefright/omx/SoftOMXPlugin.cpp
index 25b36cb7b6..6258ee08d7 100644
--- a/media/libstagefright/omx/SoftOMXPlugin.cpp
+++ b/media/libstagefright/omx/SoftOMXPlugin.cpp
@@ -64,7 +64,6 @@ static const struct {
     { "OMX.dolby.ac3.decoder", "ddpdec", "audio_decoder.ac3" },
     { "OMX.dolby.eac3-joc.decoder", "ddpdec", "audio_decoder.eac3_joc" },
     { "OMX.dolby.eac3.decoder", "ddpdec", "audio_decoder.eac3" },
-    { "OMX.dolby.ac4.decoder", "ac4dec", "audio_decoder.ac4" },
 };
 
 static const size_t kNumComponents =
